src/nvc_mount.c

```diff
@@ -404,6 +404,10 @@ symlink_libraries(struct error *err, const struct nvc_container *cnt, const char
                         /* XXX GLVND requires this symlink for indirect GLX support. */
                         if (symlink_library(err, paths[i], lib, "libGLX_indirect.so.0", cnt->uid, cnt->gid) < 0)
                                 return (-1);
+                } else if (str_has_prefix(lib, "libnvidia-opticalflow.so")) {
+                        /* XXX Fix missing symlink for libnvidia-opticalflow.so. */
+                        if (symlink_library(err, paths[i], "libnvidia-opticalflow.so.1", "libnvidia-opticalflow.so", cnt->uid, cnt->gid) < 0)
```

Comment:
Don't these symlinks get fixed by ldconfig? We can just mount the libraries themselves only.

Reply:
This is to fix an internal bug. ldconfig links with libnvidia-opticalflow.so.1 but some optics applications expects libs to be libnvidia-opticalflow.so similar to libcuda.so.